.mailmap

```diff
@@ -315,9 +340,12 @@ Josh Holmer <jholmer.in@gmail.com>
 Josh Stone <cuviper@gmail.com> <jistone@redhat.com>
 Josh Stone <cuviper@gmail.com> <jistone@fedoraproject.org>
 Julia Ryan <juliaryan3.14@gmail.com> <josephryan3.14@gmail.com>
+Jubilee Young <workingjubilee@gmail.com> <46493976+workingjubilee@users.noreply.github.com>
+Jubilee Young <workingjubilee@gmail.com> <workingjubilee@gmail.com>
```

Comment:
The two emails here are the same, is this deliberate?

Reply:
Actually now I notice there are *a lot* of duplicates in this file :thinking: 

Does anybody know why?

Reply:
I'm not sure what you mean by duplicates. Many lines that may seem like duplicates are actually necessary because of the arcane mailmap format